src/neo/SmartContract/Native/NativeContract.cs

```diff
@@ -69,6 +69,10 @@ public abstract class NativeContract
         /// Gets the instance of the <see cref="Native.NameService"/> class.
         /// </summary>
         public static NameService NameService { get; } = new();
+        /// <summary>
+        /// Gets the instance of the <see cref="NotaryContract"/> class.
+        /// </summary>
+        public static NotaryContract Notary { get; } = new();
```

Comment:
Can we open a bug in devpack repo to track adding NotaryContract to SmartContract Framework?

Reply:
https://github.com/neo-project/neo-devpack-dotnet/pull/556